src/main/resources/db/migration/V1_createTables.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS "user"(
+    "id" serial PRIMARY KEY,
+    "username" varchar(100) NOT NULL,
+    "password" varchar(255) NOT NULL,
+    "encryptionKey" varchar(255) NOT NULL,
```

Comment:
mixed casing, SQL column names should be in snake_case only

Reply:
Corrected